etc/initsystem/prepare-dirs.cmake

```diff
@@ -3,6 +3,11 @@
 # This script prepares directories and files needed for running Icinga2
 #
 
+# With sysvinit we need to pass the environment this way. Or do we?
+if [ "$1" != "" ]; then
+	source "$1"
+fi
+
```

Comment:
This does only check whether a parameter was given. v2.8.4 checks if the file actually exists.

https://github.com/Icinga/icinga2/blob/78322b095310bb95c2015ea03457c52d327991e8/etc/initsystem/prepare-dirs#L6

Reply:
Good point